style2.css

```diff
@@ -126,3 +126,78 @@
   margin-top: -50px;
   margin-bottom: 24px;
 }
+
+/* Form section */
+
+.get-in-touch {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin: 59px 24px;
+}
+
+.full-name {
+  box-sizing: border-box;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  background: #fff;
+  margin-left: 24px;
+  margin-right: 24px;
+  height: 48px;
+  width: 327px;
```

Comment:
- [ ] [OPTIONAL] Please add some padding so that the text input looks more like the design.

Reply:
Thanks for the nice suggestion, I have implemented this.